activesupport/lib/active_support/core_ext/time/conversions.rb

```diff
@@ -3,28 +3,28 @@
 
 class Time
   DATE_FORMATS = {
-    :db           => '%Y-%m-%d %H:%M:%S',
-    :number       => '%Y%m%d%H%M%S',
-    :nsec         => '%Y%m%d%H%M%S%9N',
-    :time         => '%H:%M',
-    :short        => '%d %b %H:%M',
-    :long         => '%B %d, %Y %H:%M',
-    :long_ordinal => lambda { |time|
+    db:            '%Y-%m-%d %H:%M:%S',
+    number:        '%Y%m%d%H%M%S',
+    nsec:          '%Y%m%d%H%M%S%9N',
+    time:          '%H:%M',
+    short:         '%d %b %H:%M',
+    long:          '%B %d, %Y %H:%M',
+    long_ordinal:  lambda { |time|
       day_format = ActiveSupport::Inflector.ordinalize(time.day)
       time.strftime("%B #{day_format}, %Y %H:%M")
     },
-    :rfc822       => lambda { |time|
+    rfc822:        lambda { |time|
       offset_format = time.formatted_offset(false)
       time.strftime("%a, %d %b %Y %H:%M:%S #{offset_format}")
     },
-    :iso8601      => lambda { |time| time.iso8601 }
+    iso8601:       lambda { |time| time.iso8601 }
   }
 
   # Converts to a formatted string. See DATE_FORMATS for built-in formats.
   #
   # This method is aliased to <tt>to_s</tt>.
   #
-  #   time = Time.now                    # => Thu Jan 18 06:10:17 CST 2007
+  #   time = Time.now                    # => 2007-01-18 06:10:17 -06:00
```

Comment:
This din't change anything it was just a output correction. The output given earlier was wrong